pkg/provider/aws/client.go

```diff
@@ -140,15 +139,13 @@ type AwsClient struct {
 func addProxyConfigToSessionOptConfig(config *aws.Config) {
 	awsProxyUrl := viper.GetString(ProxyConfigKey)
 	if awsProxyUrl == "" {
-		_, _ = fmt.Fprintf(os.Stderr, "[WARNING] `%s` not configured. Please add this to your osdctl configuration to ensure traffic is routed though a proxy.\n", ProxyConfigKey)
-		if skipAwsProxyCheck := viper.GetBool(SkipProxyCheckKey); !skipAwsProxyCheck {
-			_, _ = fmt.Fprint(os.Stderr, "Please confirm that you would like to continue with [y|N] ")
-			var input string
-			_, _ = fmt.Scanln(&input)
-			if strings.ToLower(input) != "y" {
-				_, _ = fmt.Fprintln(os.Stderr, "Must enter 'y' to continue; exiting...")
-				os.Exit(0)
-			}
+		_, _ = fmt.Fprintf(os.Stderr, "[ERROR] `%s` not configured. Please add this to your osdctl configuration to ensure traffic is routed though a proxy.\n", ProxyConfigKey)
```

Comment:
This still feels more like a warning to me, as we aren't crashing or hard-enforcing anything in the absence of an `aws_proxy`. I'm not fully opposed, but what made you up the severity here?

Reply:
In my head, warnings are often allowed to accumulate. Errors must be dealt with, which I'm demanding the operator does in the next couple lines of code. But I also don't really mind either way.